pelican/themes/notmyidea/static/css/main.css

```diff
@@ -176,6 +177,12 @@ div.figure p.caption, figure p.caption { /* margin provided by figure */
 
 	/* Banner */
 	#banner h1 {font-size: 3.571em; line-height: 0;}
+	@media screen and (max-width: 480px) {
```

Comment:
I'm far from being a CSS expert but seeing anything hardcoded in pixels makes me worry. I'm sure with the right font and size combination coming from user.css this is going to misbehave in certain conditions. Can you please explain why this line-height is needed at all ever?

Reply:
I've reworked this. The line-height above is really the problem, so I've changed that. The vertical spacing was also a bit off compared to the existing version so I've adjusted that to match.